consumers/src/unit-test/java/uk/gov/moj/sdt/cmc/consumers/CMCConsumerGatewayTest.java

```diff
@@ -0,0 +1,66 @@
+package uk.gov.moj.sdt.cmc.consumers;
+
+import java.io.IOException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.moj.sdt.cmc.consumers.api.IBreathingSpace;
+import uk.gov.moj.sdt.cmc.consumers.converter.XmlToObjectConverter;
+import uk.gov.moj.sdt.cmc.consumers.model.breathingspace.BreathingSpaceRequest;
+import uk.gov.moj.sdt.domain.api.IIndividualRequest;
+
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+/**
+ * Test class for the consumer gateway.
+ *
+ * @author Mark Dathorne
+ */
+@ExtendWith(MockitoExtension.class)
+class CMCConsumerGatewayTest {
+
+    private static final long CONNECTION_TIME_OUT = 10;
+
+    private static final long  RECEIVE_TIME_OUT = 10;
+
+    private static final String XML = "";
+
+    private CMCConsumerGateway cmcConsumerGateway;
+
+    private XmlToObjectConverter xmlToObject;
+
+    private IBreathingSpace breathingSpace;
+
+    private IIndividualRequest individualRequest;
+
+    @BeforeEach
+    public void setUpLocalTests() {
+        xmlToObject = mock(XmlToObjectConverter.class);
+        breathingSpace = mock(IBreathingSpace.class);
+        individualRequest = mock(IIndividualRequest.class);
```

Comment:
Could use Mock annotation on the class fields instead of explicitly setting up mocks here.

Reply:
done